Jointify/Views/Screens/ProcessingView.swift

```diff
@@ -18,6 +18,8 @@ struct ProcessingView: View {
     
     // MARK: State Instance Propoerties
     @State private var progress: Int = 0
+    @State private var removedFramesCounter = 0
+    private let acceptedFramesThreshold: Double = 0.7
```

Comment:
Not a state variable, move into `// MARK: Stored Instance Properties`.

Reply:
I think this actually has to be a state variable, since every view is a struct, and variables are immutable in a struct (unless you include @state before them).
Tested it and I get error messages, if I dont include @state before, saying that the variable is immutable.
Agree?

Reply:
Ah wow, that's interesting. Looks good with the nice `MARKs` then 👍